functions-app/db-check.js

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
```

Comment:
this has been duplicated twice now, any chance we can source from root, or am i right in assuming that's not possible due to node/yarn sourcing rules?

Reply:
If there is a nice solution then let me know.  

`/mtc` is mounted so the whole project is made available - however, the only dependency we need is `.env`.  Adding more dependencies into '/mtc' would slightly complicate using other methods in the future, e.g. mutagen.

However, the node script has it's own dependencies: `mssql` and our own `config.js`, so in de-duplicating a small amount of code, we would need to duplicate the larger quantity of dependencies.  

Perhaps there is a better way?  If not, let's leave it for now.

Reply:
sounds like we should stick with your approach 👍